src/debug/index.ts

```diff
@@ -212,7 +212,12 @@ function logUnit(unit: Unit<any>, config?: Config) {
         logTrace(store);
       }
     });
-    unit.onCreateEffect(logEffect);
+    unit.onCreateEffect(effect => {
+      logEffect(effect)
```

Comment:
`logEffect` only attaches logs to `done` and `fail` - classic logs for `fx` calls are still missing and it can be visible in the test too
There is a log of `fx` itself missing:
```suggestion
      log(effect, 'effect');
      logEffect(effect);
```

Reply:
done!
But notice it affected a bit more tests